### PR TITLE
Feature/hamburger menu logic

### PR DESCRIPTION
**Pull request by JoannaLapa**
A toggled menu when a button is clicked with a blocked scroll on the body and an overlay when a sidebar is shown.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,6 @@
 
 <head>
   <meta charset="UTF-8">
-  <link rel="icon" href="/favicon.ico">
```
Comment (ale-krecisz) [inline index.html:6]: Why did you remove favicon? You don't want any?

File: src/components/basicComponents/TheHeader.vue [hunk 2]
```diff
@@ -3,19 +3,22 @@
...
+          <button class="cursor-pointer" @click="toggleMenu">
+            <IconOpenMenu v-if="isOpened" />
+            <IconCloseMenu v-else />
+          </button>
         </div>
-        <div class="relative right-0 z-20 flex w-3/4 self-end bg-neutral-200">
-          <ul class="flex h-screen flex-col gap-11 p-6">
+        <div
+          class="relative right-0 flex w-3/4 self-end bg-neutral-200 before:absolute before:-left-2/4 before:h-screen before:w-[50%] before:bg-black before:opacity-50"
+          :class="{ hidden: isOpened }"
```
Comment (ale-krecisz) [inline src/components/basicComponents/TheHeader.vue]: check `v-show` it might be good here

File: index.html (hunk 1 above)
Comment (JoannaLapa) [reply index.html:6]: ok, I pasted it back to the project

File: src/components/basicComponents/TheHeader.vue (hunk 2 above)
Comment (JoannaLapa) [reply src/components/basicComponents/TheHeader.vue]: done ;)
